tags/tags.toml

```
@@ -841,3 +841,12 @@ content = """
 If you are using the `GUILD_MEMBERS` or `GUILD_PRESENCES` intents, you need to enable them in the developer portal:
 • [Developer Portal](<https://discord.com/developers/applications>) > Your app > Bot > Privileged Gateway Intents
 """
+
+[collection-array]
+keywords = ["collection-array", "array", "array()", "collection array"]
```

```suggestion
keywords = ["collection-array", "collection array"]
```

Keep the keywords free of extra symbols, if possible (and `array` is too generic for what the tag conveys, this is not really about how arrays work, which that would suggest)